utils/post.js

```diff
@@ -333,7 +333,13 @@ const modifyNewItemAnonymity = (newItem) => {
   return newItem;
 };
 
-async function filterFeeds(userId, feeds = [], feed_id = null, threshold = null) {
+async function filterFeeds(
+  userId,
+  selfAnonymousUserId,
+  feeds = [],
+  feed_id = null,
+  threshold = null
+) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `filterFeeds` now takes an additional parameter `selfAnonymousUserId`. This is a breaking change if there are other parts of the code that call this function without passing this new argument. If this function is widely used across the codebase, consider providing a default value for `selfAnonymousUserId` to maintain backward compatibility.
```diff
-  async function filterFeeds(
-    userId,
-    selfAnonymousUserId,
-    feeds = [],
-    feed_id = null,
-    threshold = null
-  ) {
+  async function filterFeeds(
+    userId,
+    feeds = [],
+    feed_id = null,
+    threshold = null,
+    selfAnonymousUserId = null
+  ) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->